### ISSUE TITLE
Atlas Phase 2

### ISSUE THREAD

**Issue by zorg2044** [CobaltWolf/Bluedog-Design-Bureau#1551]
Things that aren't going to make the 1.14 update and are planned or being considered for a future update.

Atlas Burner 2
- [ ] New burner 2 kit

fairings
- [ ] Atlas Burner 2 fairing
- [ ] Thor Burner 2 fairing

Payloads
- [ ] MSD kit for Altair
- [ ] MSD plume deflector subsatellite

Decals
- [ ] Tracking checker
- [ ] SPP red tape
- [ ] Star stripe thing?

Extra features
- [ ] Auto close of umbilical hatch on skirt when lifting off

**Comment by JacobB094** [comment]
Is the dedicated M55 decoupler not happening after all? It'd have been a nice addition. It could have also been used for SOLTAN boosters, and for crazy designs like MDD Barbarian, which use a radially attached Delta core.

**Comment by zorg2044** [comment]
> Is the dedicated M55 decoupler not happening after all? It'd have been a nice addition. It could have also been used for SOLTAN boosters, and for crazy designs like MDD Barbarian, which use a radially attached Delta core.

M55 is 1.25m I believe so not applicable to SOLTAN or Delta at 1.5m. The main issue with the M55 is its difficult to do visually clean attachment for such large boosters due to interference from the lox pipe on the core tank. I know its mentioned in that one poster but the whole thing looks awkward and I'm not really feeling it.

**Comment by JacobB094** [comment]
Right, mixed it up with Castor 120. Still, maybe something could be done with an offset attachment point? Either hanging a bit further out, or off to the side. It's a fairly common type of booster for various Saturn configurations, so a dedicated decoupler would be really appreciated either way.

**Comment by JacobB094** [comment]
Would the IABS be a possibility to add as part of Atlas Phase II? 
https://upload.wikimedia.org/wikipedia/commons/8/89/IABS_Illustration.png
Basically, a bunch of hypergolic tanks feeding four R-4Ds, surrounded by a solar panel shaped like a flattened ring. Goes on top of the Centaur, I think it'd work out to 1.875m outer diameter. An obscure little piece of kit, but those small kick stages are quite useful.
